app/mender.go

```diff
@@ -444,6 +444,10 @@ func (m *Mender) GetRetryPollInterval() time.Duration {
 	return t
 }
 
+func (m *Mender) GetRetryPollCount() int {
```

Comment:
You should also add this [here](https://github.com/mendersoftware/mender/blob/4e9236e404f0acc20b71ffe1b82ba556c544ab2b/app/mender.go#L38-L66) (which will probably trigger an avalanche of other places you need to add it!).

Reply:
done-done.